python_modules/dagster/dagster/_core/pipes/client.py

```diff
@@ -111,6 +111,11 @@ def get_asset_check_result(self) -> AssetCheckResult:
         return _check_result_from_pipes_results(self.get_results())
 
     def get_custom_messages(self) -> Sequence[Any]:
+        """Get the sequence of deserialized from JSON data that was reported from the external process using
```

Comment:
> deserialized from JSON data

"deserialized JSON data"? This is a bit difficult to understand - would this suggestion still be accurate? Or is there another way to make this clearer?